posts/238-new-make-shuffle-mode.md

```diff
@@ -5,9 +5,22 @@ date: February 23, 2022
 
 # TL;DR:
 
-I implemented new `--shuffle` option for `GNU make` to simulate
+I implemented new `--shuffle` option for GNU make to simulate
 non-deterministic build order in parallel makefiles.
 
+**UPDATE**: `--shuffle` mode was released as part of GNU make 4.4 and
+can be used as:
+
+- `make --shuffle ...` from the command line for interactive use.
```

Comment:
Maybe the last thing, I find "interactive use" a bit misleading here, interactive in my mind is when e.g. the software asks for confirmation or is using curses. Just my two cents.

Reply:
Yeah, I agree. Changed to just "`make --shuffle ...` from the command line.". How about that?

Reply:
@trofi seems okay :+1: